applications/volume_rendering_xr/main.cpp

```diff
@@ -169,58 +285,74 @@ class App : public holoscan::Application {
                                   {"config", required_argument, 0, 'c'},
+                                  {"write_config", required_argument, 0, 'w'},
                                   {"density", required_argument, 0, 'd'},
                                   {"mask", required_argument, 0, 'm'},
+                                  {"eye-tracking", no_argument, 0, 'e'},
                                   {0, 0, 0, 0}};
 
   // parse options
   while (true) {
     int option_index = 0;
 
-    const int c = getopt_long(argc, argv, "hc:d:m:", long_options, &option_index);
+    const int c = getopt_long(argc, argv, "hc:w:d:m:e", long_options, &option_index);
 
     if (c == -1) { break; }
 
     const std::string argument(optarg ? optarg : "");
     switch (c) {
       case 'h':
-        std::cout << "Holoscan OpenXR volume renderer."
-                  << "Usage: " << argv[0] << " [options]" << std::endl
-                  << "Options:" << std::endl
-                  << "  -h, --help                            Display this information" << std::endl
-                  << "  -c <FILENAME>, --config <FILENAME>    Name of the renderer JSON "
-                     "configuration file to load (default '"
-                  << render_config_file_default << "')" << std::endl
-                  << "  -d <FILENAME>, --density <FILENAME>   Name of density volume file to load "
-                     "(default '"
-                  << density_volume_file_default << "')" << std::endl
-                  << "  -m <FILENAME>, --mask <FILENAME>      Name of mask volume file to load "
-                     "(default '"
-                  << mask_volume_file_default << "')" << std::endl;
+        std::cout
+            << "Holoscan OpenXR volume renderer."
+            << "Usage: " << argv[0] << " [options]" << std::endl
+            << "Options:" << std::endl
+            << "  -h, --help                            Display this information" << std::endl
+            << "  -c <FILENAME>, --config <FILENAME>    Name of the renderer JSON "
+               "configuration file to load (default '"
+            << render_config_file_default << "')" << std::endl
+            << "  -w <FILENAME>, --write_config <FILENAME> Name of the renderer JSON "
+               "configuration file to write to (default '')"
+            << std::endl
+            << "  -d <FILENAME>, --density <FILENAME>   Name of density volume file to load "
+               "(default '"
+            << density_volume_file_default << "')" << std::endl
+            << "  -m <FILENAME>, --mask <FILENAME>      Name of mask volume file to load "
+               "(default '"
+            << mask_volume_file_default << "')" << std::endl
+            << "  -e, --eye-tracking                    Enable eye tracking and foveated rendering."
+            << std::endl;
         return 0;
 
       case 'c':
+        HOLOSCAN_LOG_DEBUG("Config file: " + argument);
```

Comment:
Probably some debug code, could be removed.

Reply:
Updated.